src/test/kotlin/apply/ApplicationFormFixtures.kt

```diff
@@ -1,54 +1,71 @@
 package apply
 
+import apply.application.AnswerRequest
 import apply.domain.applicationform.ApplicationForm
-import apply.domain.recruitmentitem.Answer
-import apply.domain.recruitmentitem.Answers
+import apply.domain.applicationform.ApplicationFormAnswer
+import apply.domain.applicationform.ApplicationFormAnswers
+
+private const val LONG_LENGTH = 2000
 
 fun createApplicationForm(
     applicantId: Long = 1L,
     recruitmentId: Long = 1L,
     referenceUrl: String = "http://example.com",
-    answers: Answers = createAnswers()
+    applicationFormAnswers: ApplicationFormAnswers = createApplicationFormAnswers()
 ): ApplicationForm {
-    return ApplicationForm(applicantId, recruitmentId, referenceUrl, answers)
+    return ApplicationForm(applicantId, recruitmentId, referenceUrl, applicationFormAnswers)
 }
 
-fun createAnswers(
-    answers: MutableList<Answer> = mutableListOf(
-        createAnswer(),
-        createAnswer("책임감", 2L)
+fun createApplicationFormAnswers(
+    applicationFormAnswers: MutableList<ApplicationFormAnswer> = mutableListOf(
+        createApplicationFormAnswer(),
+        createApplicationFormAnswer("책임감", 2L)
     )
-): Answers {
-    return Answers(answers)
+): ApplicationFormAnswers {
+    return ApplicationFormAnswers(applicationFormAnswers)
 }
 
-fun createAnswer(
+fun createApplicationFormAnswer(
     contents: String = "스타트업을 하고 싶습니다.",
     recruitmentItemId: Long = 1L
-): Answer {
-    return Answer(contents, recruitmentItemId)
+): ApplicationFormAnswer {
+    return ApplicationFormAnswer(contents, recruitmentItemId)
+}
+
+fun createAnswerRequest(
+    contents: String = "책임감",
+    recruitmentItemId: Long = 1L
+): AnswerRequest {
+    return AnswerRequest(contents, recruitmentItemId)
+}
+
+fun createExcessOfLengthAnswerRequest(
+    contents: String = "*".repeat(LONG_LENGTH),
```

Comment:
```suggestion
    excessOfLengthContents: String = "*".repeat(LONG_LENGTH),
```

요런 느낌으로 초과된 문자열임을 표시해줘도 좋을 것 같아요.
변수명이 너무 길어서 다른 네이밍도 좋구요.

혹은 `contents`를 놔두고 뒤에 `LONG_LENGTH`에 의미를 줘도 되겠네요.

Reply:
아무래도 Dto와 필드명이 다르면 헷갈릴 수 있을 것 같아서 `LONG_LENGTH` 를 `EXCESS_LENGTH`로 수정했습니다 :)